.github/workflows/publish-wiki.yml

```diff
@@ -0,0 +1,23 @@
+---
+name: Publish wiki
+
+on:
+  push:
+    branches: [main]
+    paths:
+      - wiki/**
```

Comment:
Should this be renamed to docs folder instead of wiki? Seems more generic @prudhvigodithi @peterzhuamazon ?

Reply:
Hi @chawinphat 

Can you please rename the folder to `docs` so it can be generic and multi-used too.

Reply:
Updated.

Reply:
```suggestion
      - docs/**
```

Reply:
Hi @chawinphat Can you change the path here? Thanks!